CIScripts/Tests/AgentTests.ps1

```diff
@@ -1,12 +1,33 @@
+$DefineTestIfAnOutputSuggestsThatAllTestsHavePassed = {
+    function Test-IfAnOutputSuggestsThatAllTestsHavePassed {
```

Comment:
[grammar-nazi] I think "An" is not required here
Also this is google test specific so I think the function name should reflect it. Maybe something like
`Test-IfGTestStdoutSaysPassed`?